api/src/db/db.service.ts

```diff
@@ -461,12 +459,14 @@ export class DbService extends EventEmitter {
             }
 
             // Always include _id and type
-            if (doc1._id === doc2._id) {
-                acc._id = doc1._id;
-            }
-            if (doc1.type === doc2.type) {
-                acc.type = doc1.type;
+            acc._id = doc1._id;
+            acc.type = doc1.type;
+
+            // Include parentType if it exists
+            if (doc1.parentType) {
+                acc.parentType = doc1.parentType;
             }
```

Comment:
Do we indeed need type and parentType in the diff? Then we would need to exclude those when rendering a visual change diff in the CMS.

Reply:
Removed parentType field, but added the same logic to make sure the parentID are always included in diff's